packages/web-twig/src/Resources/components/Spinner/Spinner.twig

```diff
@@ -10,7 +9,7 @@
 {# Miscellaneous #}
 {%- set _styleProps = useStyleProps(props) -%}
 {%- set _classNames = [ _rootClassName, _rootAnimationClassName, _styleProps.className ] -%}
-{%- set _mainPropsWithoutLocal = props | filter((value, prop) => prop is not same as('color') and prop is not same as('class')) -%}
+{%- set _mainPropsWithoutLocal = props | filter((value, prop) => prop is not same as('color')) -%}
```

Comment:
```suggestion
{%- set _mainPropsWithoutColor = props | filter((value, prop) => prop is not same as('color')) -%}
```

We are filtering out only one prop so I think it would be better this way for naming or use the `_mainPropsWithoutSpinnerProps/_mainPropsWithoutLocalApi/_mainPropsWithoutComponentProps`. Yeah, I have a problem with `Local` 🍻  :-)

Reply:
Fixed.